boards/nxp/mimxrt1170_evk/board.cmake

```diff
@@ -30,6 +30,11 @@ elseif(CONFIG_SOC_MIMXRT1176_CM4)
  board_runner_args(linkserver "--core=cm4")
 endif()
 
+if(NOT SYSBUILD)
+board_runner_args(jlink  "--no-reset")
+board_runner_args(linkserver  "--no-reset")
```

Comment:
2 space indent for cmake

Reply:
Double check on this, you did mean for me to put two spaces to indent these lines correct? The rest of the file is using a single space but due to the suggestion I used double space here, I may need to update the rest of the file in the future.

Reply:
2 space **indent** for cmake e.g.:
```
if(BLAH)
  set()
```
1 space **delimiter** for cmake e.g.:
```
set(my_var "my_val")
```

Reply:
In that case, my recent update fixed this, but I may need to update the rest of the file in a different PR.